libraries/Microsoft.Bot.Builder.Azure/CosmosDbPartitionedStorage.cs

```diff
@@ -216,13 +231,14 @@ private async Task InitializeAsync()
 
                 if (_container == null)
                 {
-                    _container = await _client
+                    var containerResponse = await _client
                         .GetDatabase(_cosmosDbStorageOptions.DatabaseId)
-                        .CreateContainerIfNotExistsAsync(
-                            _cosmosDbStorageOptions.ContainerId,
-                            DocumentStoreItem.PartitionKeyPath,
-                            _cosmosDbStorageOptions.ContainerThroughput)
+                        .DefineContainer(_cosmosDbStorageOptions.ContainerId, DocumentStoreItem.PartitionKeyPath)
+                        .WithIndexingPolicy().WithAutomaticIndexing(false).WithIndexingMode(IndexingMode.None).Attach()
+                        .CreateIfNotExistsAsync()
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos.database.definecontainer?view=azure-dotnet#Microsoft_Azure_Cosmos_Database_DefineContainer_System_String_System_String_

```suggestion
                        .CreateIfNotExistsAsync(_cosmosDbStorageOptions.ContainerThroughput)
```

Reply:
@j82w Ah! Thank you! I will update.